samples/android-integration/launching-activities/src/main/java/com/badoo/ribs/samples/android_integration/launching_activities/rib/builder/LaunchingActivitiesChildBuilder.kt

```diff
@@ -2,22 +2,29 @@ package com.badoo.ribs.samples.android_integration.launching_activities.rib.buil
 
 import com.badoo.ribs.builder.SimpleBuilder
 import com.badoo.ribs.core.modality.BuildParams
+import com.badoo.ribs.samples.android_integration.launching_activities.R
 import com.badoo.ribs.samples.android_integration.launching_activities.rib.LaunchingActivities
 import com.badoo.ribs.samples.android_integration.launching_activities.rib.LaunchingActivitiesInteractor
 import com.badoo.ribs.samples.android_integration.launching_activities.rib.LaunchingActivitiesNode
+import com.badoo.ribs.samples.android_integration.launching_activities.rib.LaunchingActivitiesView
 import com.badoo.ribs.samples.android_integration.launching_activities.rib.LaunchingActivitiesViewImpl
 
-class LaunchingActivitiesBuilder(
+class LaunchingActivitiesChildBuilder(
         private val dependency: LaunchingActivities.Dependency
 ) : SimpleBuilder<LaunchingActivities>() {
     override fun build(buildParams: BuildParams<Nothing?>): LaunchingActivities {
         val interactor = LaunchingActivitiesInteractor(
                 buildParams = buildParams,
                 activityStarter = dependency.activityStarter
         )
+        val viewDependency = object : LaunchingActivitiesView.Dependency {
+            override fun getTitleResource(): Int = R.string.child_title
+            override fun getDescriptionResource(): Int = R.string.child_description
+        }
+
         return LaunchingActivitiesNode(
                 buildParams = buildParams,
-                viewFactory = LaunchingActivitiesViewImpl.Factory().invoke(null),
+                viewFactory = LaunchingActivitiesViewImpl.Factory(layoutRes = R.layout.rib_launching_activities).invoke(viewDependency),
```

Comment:
Don't override default layout here. If anywhere, then in customisation, but we don't have that in this sample (which is alright).

For the same reason, maybe extract the view factory instance itself to a field, as it usually comes from customisation, not created on the fly.

Reply:
I have changed the rib hierarchy so overriding layouts is not needed anymore.